server/package.json

```diff
@@ -26,16 +27,20 @@
   "dependencies": {
     "@hapi/joi": "^17.1.1",
     "@sendgrid/mail": "^7.2.4",
+    "@types/inquirer": "^7.3.1",
     "africastalking": "^0.4.5",
     "argon2": "^0.26.2",
     "axios": "^0.19.2",
+    "commander": "^6.1.0",
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "express-rate-limit": "^5.1.3",
     "google-auth-library": "^6.0.2",
     "helmet": "^3.22.0",
+    "inquirer": "^7.3.3",
     "mongodb": "^3.5.5",
+    "ts-node": "^9.0.0",
```

Comment:
Do we need `ts-node`? I assume it won't be used in the built version. If it's just used during development, move it `devDependencies`, if it's not used at all, uninstall it.